packages/webpack-cli/.eslintrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
     root: false,
-    parserOptions: { ecmaVersion: 2020, sourceType: 'module' },
+    parserOptions: { ecmaVersion: 2020, sourceType: 'script' },
```

Comment:
script is default, removing also works

Reply:
Default root config contains `parserOptions: { ecmaVersion: 2020, sourceType: 'module' },` so I override it, in future we should revisit eslint configuration, I think now we can avoid it to continue development

Reply:
Ah yes, good catch!